web/src/poll_widget.ts

```diff
@@ -176,23 +206,23 @@ export function activate({
         });
     }
 
-    function check_option_button() {
+    function check_option_button(): void {
         const $poll_option_input = $elem.find("input.poll-option");
-        const option = $poll_option_input.val().trim();
+        const option = ($poll_option_input.val() as string)?.trim();
```

Comment:
Never use unchecked casts for this. Instead, give TypeScript the information it needs to understand that the value already has the right type.

```ts
        const $poll_option_input = $elem.find<HTMLInputElement>("input.poll-option");
        const option = $poll_option_input.val()!.trim();
```